hoomd/pytest/test_snapshot.py

```diff
@@ -125,6 +126,131 @@ def test_configuration(s):
         assert s.configuration.dimensions == 3
 
 
+def generate_outside(box, interior_points, multipliers, initial_images):
+    """Generate test cases from interior points by adding box vectors."""
+    box = Box.from_box(box)
+    lattice_vectors = box.lattice_vectors
```

Comment:
I looked into the difference between `lattice_vectors` and `matrix` and filed an issue (#1070). Since this is using the transpose, it can be replaced by the matrix.
```suggestion
    matrix = box.matrix
```